### PR TITLE
install / uninstall utilities

### PR DESCRIPTION
**Pull request by saasglue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+
+### Install
+1. Activate the conda environment where you want to install langchain
+2. Run the install script
+```
+$ ./bin/install.sh
+```
+
+### Uninstall
+1. Active the conda environment where you want to uninstall langchain
```
Comment (jkndrkn) [inline README.md:13]: s/Active/Activate

File: README.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+2. Run the uninstalls script
```
Comment (jkndrkn) [inline README.md:14]: s/uninstalls/uninstall

**Review by jkndrkn — COMMENTED** [review]
Looks good. Found a couple typos.

**Comment by saasglue** [comment]
Ready for final review/merge

**Pull request merged**